css/src/atomics/colors.scss

```diff
@@ -150,6 +154,16 @@
 	}
 }
 
+.background-color-body-learn {
+	outline-color: $text !important;
+	background-color: $body-background-learn !important;
+}
+
+.background-colo-card {
```

Comment:
Given Atlas doesn't contain a card component, I think this one shouldn't be added here

Reply:
Yeah, I was thinking if it's a good idea since the component is not there. Will remove

Reply:
Removed